src/views/network/CreateNetwork.vue

```diff
@@ -95,8 +95,9 @@ export default {
     isAdvancedZoneWithoutSGAvailable () {
       for (const i in this.actionZones) {
         const zone = this.actionZones[i]
-        if (zone.networktype === 'Advanced' && zone.securitygroupsenabled !== true) {
-          return true
+        // Dont display isolated networks for advanced with sg enabled
+        if (zone.networktype === 'Advanced' && zone.securitygroupsenabled === true) {
+          return false
```

Comment:
Would this break if there are multiple zones and one with SG Enabled ?

Reply:
Fixed it to match the zone id.